pipes/WDL/workflows/tasks/metagenomics.wdl

```diff
@@ -0,0 +1,111 @@
+
+# TO DO:
+# kraken_build (input & output tarballs)
+# diamond, bwa, etc
+
+task kraken {
+  Array[File] reads_unmapped_bam
+  File        kraken_db_tar_lz4
+
+  parameter_meta {
+    kraken_db_tar_lz4:  "stream" # for DNAnexus, until WDL implements the File| type
+    #reads_unmapped_bam: "stream" # for DNAnexus, until WDL implements the File| type
+  }
+
+  command {
+    set -ex -o pipefail
+
+    if [ -d /mnt/tmp ]; then
+      TMPDIR=/mnt/tmp
+    fi
+    DB_DIR=$(mktemp -d)
+
+    # decompress DB to $DB_DIR
+    cat ${kraken_db_tar_lz4} |
+      read_utils.py extract_tarball \
+        - $DB_DIR \
+        --pipe_hint=${kraken_db_tar_lz4} \
+        --loglevel=DEBUG
+
+    ## prep input and output file names
```

Comment:
Can we try bringing the multiple processing back?

Reply:
Oh yeah, let me give that a try. I'd have to test it manually, since this current framework doesn't yet execute any of the WDL pipelines, but will do so.